R/ons_uk_population_2023.R

```diff
@@ -0,0 +1,65 @@
+#' ONS Mid-2023 Population Estimate for UK
+#'
+#' ONS Population Estimates for Mid-year 2023
+#' National and subnational mid-year population estimates for the UK and its
+#'  constituent countries by administrative area, age and sex (including
+#'   components of population change, median age and population density).
+#'
+#' Data sourced from
+#' \href{https://www.ons.gov.uk/peoplepopulationandcommunity/populationandmigration/populationestimates/datasets/populationestimatesforukenglandandwalesscotlandandnorthernireland}
+#'
+#' @docType data
+#'
+#' @keywords datasets ons population
+#'
+#' @format Tibble with six columns
+#' \describe{
+#' \item{sex}{male or female}
+#' \item{Code}{The country/geography code}
+#' \href{Name}{country of the UK}
+#' \item{Geography}{Country}
+#' \item{age}{year of age}
+#' \item{count}{the number of people in this group}
+#' }
+#'
+#' @source \href{https://www.ons.gov.uk/peoplepopulationandcommunity/populationandmigration/populationestimates/datasets/populationestimatesforukenglandandwalesscotlandandnorthernireland}{ONS Estimates of the population for the UK, England, Wales, Scotland, and Northern Ireland}
+#'
+#' @usage data(ons_uk_population_2023)
+#'
+#' @examples
+#' data(ons_uk_population_2023)
+#'
+#' library(readxl)
+#' library(tidyverse)
+#' library(tidyr)
+
+#' Load the data in
```

Comment:
```suggestion
#' # Load the data in
```

that line needs to be forced to be a comment: it's trying to run `Load the data in` as an R command

Reply:
actually, I would remove and change the entire example here:

* the example is meant to be about using the data, not how to generate it
* the example includes a path to a file on your local computer

Reply:
Yeah, thats all been fixed just now anyway, but still failing !